code/game/objects/structures/safe.dm

```diff
@@ -6,29 +6,44 @@ FLOOR SAFES
 
 //SAFES
 /obj/structure/safe
-	name = "safe"
-	desc = "A huge chunk of metal with a dial embedded in it. Fine print on the dial reads \"Scarborough Arms - 2 tumbler safe, guaranteed thermite resistant, explosion resistant, and assistant resistant.\""
+	name = "\improper Safe"
+	desc = "A huge chunk of metal with a dial embedded in it. Fine print on the dial reads \"Nanotrasen Arms - 2 tumbler safe, guaranteed thermite resistant, explosion resistant, and assistant resistant.\""
 	icon = 'icons/obj/structures.dmi'
 	icon_state = "safe"
 	anchored = 1
 	density = 1
-	var/open = 0		//is the safe open?
+	var/open = FALSE	//is the safe open?
+	var/unlocked = FALSE

```

Comment:
I would prefer you flip this on its head and do `var/locked = TRUE` etc... because most of the rest of the codebase uses locked rather than unlocked everywhere. As far as I can tell anyway. Best to keep it consistent.